public/omni/getting-started/install-and-configure-omnictl.mdx

```diff
@@ -54,24 +88,41 @@ If you would like to merge the `omniconfig.yaml` with an existing configuration,
 omnictl config merge ./omniconfig.yaml
 ```
 
-List the contexts to verify that the `omniconfig` was added:
+You can confirm that the new configuration was added successfully by listing your contexts with this command:
 
-```bash
-$ omnictl config contexts
-CURRENT   NAME         URL
-          ...
-          example      https://example.omni.siderolabs.io/
-          ...
-```
+<Tabs>
+  <Tab title= "Command">
+
+    ```bash
+    omnictl config contexts
+    ```
+  </Tab>
+  <Tab title= "Output">
 
-Run `omnictl` for the first time to perform initial authentication using a web browser:
+    ```bash
+    CURRENT   NAME         URL
+              ...
+              example      https://example.omni.siderolabs.io/
+              ...
+    ```
+  </Tab>
+</Tabs>
+
+## Step 3: Authenticate and verify
+
+Once installed and configured, the last step is authentication.
+This connects your local CLI to your Omni account.
+
+Run the following command to trigger authentication:
 
 ```bash
 omnictl get clusters
 ```
 
-If the browser window does not open automatically, it can be opened manually by copying and pasting the URL into a web browser:
+A browser window should open automatically for sign-in. If it doesn’t, copy and paste the URL manually:
 
 ```bash
 BROWSER=echo omnictl get clusters
```

Comment:
@Unix4ever isn't there a line like `--skip-open-browser` they should add to their omni config to make it not open the browser automatically?